replica/compaction_group.hh

```diff
@@ -135,6 +135,14 @@ public:
     }
 };
 
+class compaction_group_manager {
+public:
+    virtual ~compaction_group_manager() {}
+    virtual std::vector<std::unique_ptr<compaction_group>> make_compaction_groups() const = 0;
+    virtual size_t compaction_group_of(dht::token) const = 0;
+    virtual size_t x_log2_compaction_groups() const = 0;
```

Comment:
why "x_log2" and not "log2"? What does "x" stand for? The returned value is log_2(nr compaction groups).